amanuensis/blueprints/admin.py

```diff
@@ -267,11 +267,41 @@ def update_project_state():
     )
 
 
+@blueprint.route("/projects/date", methods=["PATCH"])
+@check_arborist_auth(resource="/services/amanuensis", method="*")
+# @debug_log
+def override_project_date():
+    """
+    Updates the update_date of a project.
+    """
+
+    project_id = request.get_json().get("project_id", None)
+    year = request.get_json().get("year", 0)
+    month = request.get_json().get("month", 0)
+    day = request.get_json().get("day", 0)
+    hour = request.get_json().get("hour", 0)
+    minute = request.get_json().get("minute", 0)
+    second = request.get_json().get("second", 0)
+    microsecond = request.get_json().get("microsecond", 1)
```

Comment:
We don't need to be more granular than the day, so day, month and year are going to be enough.